plugins/pulp_docker/plugins/importers/upload.py

```diff
@@ -409,7 +409,14 @@ def process_main(self, item=None):
             if not checksum_type:
                 # Never assume. But oh well
                 checksum_type = "sha256"
-            blob_src_path = os.path.join(self.get_working_dir(), checksum + '.tar')
+
+            blob_src_path = os.path.join(self.get_working_dir(), checksum)
+            version_file_path = os.path.join(self.get_working_dir(), 'version')
+            transport_version = transport.Version.from_file(version_file_path)
```

Comment:
i was just wondering, what would be wrong with the approach of getting the version like this https://github.com/pulp/pulp_docker/pull/235/files#diff-bd957746355aec66e333c744e73a128aR30 since it's the file that consists from 1 line and then just converting that to float?

Reply:
The problem is we need integer comparison for the number after the decimal. `1.19 > 1.2`.